package_reloader.py

```diff
@@ -28,6 +33,19 @@ def relative_to_spp(path):
         for sp in [spp, spp_real]:
             if p.startswith(sp + os.sep):
                 return p[len(sp):]
+
+    if not sys.platform.startswith("win"):
+        # we try to follow symlink if the real file is not located in spp
+        for p in [path, casedpath(os.path.realpath(path))]:
```

Comment:
If you make this a set literal, a potential duplicate path on non-Windows will automatically be deduplicated. Unless order is important.